### DISCUSSION TITLE
Tidak mau membaca dotenv

### DISCUSSION THREAD

**Discussion by danteistanto** [teamalgoritma/community#364] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/112854600/193195651-f59781f8-7f2c-449a-bb79-2a758d24d858.png)
Selamt siang, symengalami kendala dotenv tidak terbaca,  mohon bantuannya. terima kasih

**Comment by kevbow** [comment, marked as answer]
Halo @danteistanto, 
Apakah sudah melakukan instalasi library yang terdapat di requirements.txt? 
Seharusnya library dotenv adalah dependencies dari salah satu library yang tertera di sana.

Jika memang sudah, kemungkinan karena Anda menggunakan cmd tetatpi belum berhasil melakukan koneksi ke virtual environment. Jika hal ini memang yang terjadi, silakan restart vscode lalu pilih kembali terminal command prompt seperti gambar di bawah ini,
![image](https://user-images.githubusercontent.com/68848168/193208937-51288737-6178-42ee-9342-4cef263ff948.png)
Setelah terminal cmd terbuka, seharusnya virtual environment yang dipilih di interpreter akan langsung teraktivasi seperti gambar di bawah ini
![image](https://user-images.githubusercontent.com/68848168/193209772-291066ed-1da3-4e53-9e51-c17811deba0f.png)
tetapi jika tidak, Anda bisa melakukan aktivasi virtual environment manual sama seperti yang dilakukan di anaconda prompt.

Demikian yang dapat saya sampaikan, semoga membantu. Terima kasih

**Reply by danteistanto** [reply]
Baik kak, terima kasih masukan nya. Akan saya coba
